Makefile

```diff
@@ -51,8 +51,7 @@ build-flow-typings:
 	$(NODE) packages/babel-types/scripts/generators/flow.js > packages/babel-types/lib/index.js.flow
 
 build-typescript-typings:
-	$(NODE) packages/babel-types/scripts/generators/typescript.js > packages/babel-types/lib/index.d.ts
-	$(NODE) packages/babel-types/scripts/generators/typescript.js --ts3.7 > packages/babel-types/lib/index-ts3.7.d.ts
+	echo "TODO: maybe add types for typescript 3.7"
```

Comment:
We can add the `index-ts3.7.d.ts` to `/src` and keep it for backward compatibilities. Just adding a note that this file is no longer updated to reflect most recent changes in `@babel/types` -- The interface definition may be incomplete but it should be good if you don't use the new features of `@babel/types`.

Folks can pin Babel types for TypeScript 3.7 support or update babel types and typescript to 4.0 as well.

Reply:
other option would be to bundle generated type definitions into single file using something like [rollup-plugin-dts](https://github.com/Swatinem/rollup-plugin-dts), and then to transpile it to older typescript using  [downlevel-dts](https://github.com/sandersn/downlevel-dts)

but - I think, this is not something needed - if updating babel? why not update typescript? (especially considering that types are more complete using it)

typically typescript is very easy to update, and honestly - I do not really imagine there is a good case when babel is to be updated, while keeping old typescript.

my preference would be - to support only the last typescript version, and to not complicate things

Reply:
> I do not really imagine there is a good case when babel is to be updated, while keeping old typescript.

Usually Babel is installed with `^` in `package.json`, while different projects install TS with `~` because it doesn't follow semver.

If you had this `package.json` from last year:
```js
"@babel/types": "^7.6.0",
"typescript": "~3.7.0`
```
it would now be broken :shrug:

I agree with @JLHwung: we can stop generating `index-ts3.7.d.ts` (we can just hard-code it and stop updating it), but we cannot remove it in a minor version.